zc_common/jwt_auth/authentication.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, **kwargs):
             setattr(self, kwarg, kwargs[kwarg])
 
     def is_authenticated(self):
-        return 'user' in self.roles or 'service' in self.roles
+        return True
```

Comment:
Django's `.is_authenticated()` method is meant to differentiate from`models.User` from `models.Anonymous` objects. That method on each object returns either `True` or `False`, respectively ([django docs](https://docs.djangoproject.com/en/1.9/ref/contrib/auth/#django.contrib.auth.models.User.is_authenticated)). So as far as maintaining similarity is concerned, we should be returning `True`. I agree that this method/function makes no difference in our case but deleting it would not be a good idea either.